raiden/utils/__init__.py

```diff
@@ -101,6 +102,23 @@ def address_decoder(addr: str) -> typing.Address:
     return addr
 
 
+def address_checksum_and_decode(addr: str) -> typing.Address:
+    """ Accepts a string address and turns it into binary.
+
+        Makes sure that the string address provided starts is 0x prefixed and
+        checksummed according to EIP55 specification
+    """
+    if addr[:2] != '0x':
```

Comment:
[`is_0x_prefixed`](https://github.com/ethereum/eth-utils#is_0x_prefixedvalue---bool) would be more readable here

Reply:
Thanks paul. Will add them in an upcoming PR.